doc/en/fcrontab.1.sgml

```diff
@@ -53,33 +53,41 @@ A copy of the license is included in gfdl.sgml.
 	    linkend="fcron.8">&fcron;(8)</link> daemon. As &fcron; internally
 	    uses a non-human readable format (this is needed because &fcron;
 	    saves more informations than the user gives, for example the time
-	    and date of next execution), the user cannot edit directly his
-	    &fcrontabf; (the one used by &fcron;).</para> <para>When a user
+	    and date of next execution), the user cannot edit their &fcrontabf;
+		directly (the one used by &fcron;).</para> <para>When a user
 	    installs a &fcrontabf;, the source file is saved in the spool
-	    directory (<filename>&fcrontabsdir;</filename>) to allow future
-	    editions, and a formatted file is generated for the &fcron; daemon,
-	    which is signaled once about ten seconds before the next minute for
-	    all changes made previously. The daemon is not informed of the changes
-	    immediately but at most once a minute
+	    directory (<filename>&fcrontabsdir;</filename>) to allow it to be
+	    edited, and a formatted file is generated for the &fcron; daemon.
+		This formatted (non human readable) file with all recent changes
+		included is then sent to the daemon.
+		The daemon is not informed of the changes immediately but after some time
 	    to keep ill disposed users from blocking the daemon by installing
-	    &fcrontabf;s over and over (ie. denial of service attack). We will call
+	    &fcrontabf;s over and over (i.e. denial of service attack).
+		By default, the file is sent once per minute (about ten seconds before
+		the full minute) to the daemon, the parameter
```

Comment:
you should say the "build time parameter" to prevent confusion

Reply:
done